custom_components/tuya_local/devices/beca_zht002_thermostat_c.yaml

```diff
@@ -0,0 +1,93 @@
+name: Zigbee Thermostat
+products:
+  - id: bf63833d92f517329fqkhz
+    name: MOES Zigbee Thermostat
+primary_entity:
+  entity: climate
+  icon: "mdi:thermostat"
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: "heat_cool"
+    - id: 16
+      name: temperature
+      type: integer
+      unit: C
+      range:
+        min: 1
+        max: 45
+      mapping:
+        - scale: 1
+    - id: 24
+      name: current_temperature
+      type: integer
+      unit: C
+      mapping:
+        - scale: 10
+    - id: 36
+      name: hvac_action
+      type: string
+      mapping:
+        - dps_val: open
+          value: heating
+        - dps_val: close
+          value: idle
+
+secondary_entities:
+  - entity: lock
+    name: Child lock
+    icon: "mdi:hand-back-right-off"
+    category: config
+    dps:
+      - id: 40
+        type: boolean
+        name: lock
+  - entity: climate
```

Comment:
Change the entity to select, and the name of the preset_mode dps below to option.